barbarik.py

```diff
@@ -171,10 +171,10 @@ def getSolutionFromQuickSampler(inputFile, numSolutions, indVarList):
 
     @staticmethod
     def getSolutionFromUniform(inputFile, numSolutions):
-        return SolutionRetriver.getSolutionFromMUSE(inputFile, numSolutions)
+        return SolutionRetriver.getSolutionFromSPUR(inputFile, numSolutions)
```

Comment:
Wait, what's going on here? Was this a bug? Also, this should be a separate change. This has nothing to do with seeds.

Reply:
We are using SPUR as the uniform sampler, not MUSE.

Reply:
Oh, I see! Can you make this into a separate pull request, I'd love to merge this in!